numpy/distutils/command/build.py

```diff
@@ -16,8 +16,8 @@ class build(old_build):
     user_options = old_build.user_options + [
         ('fcompiler=', None,
          "specify the Fortran compiler type"),
-        ('parallel=', 'j',
```

Comment:
This change seems unrelated, why does `parallel` need to be removed here and in `finalize_options`?

Reply:
It is redundant - the base `build` class has this option (on python 3)

Reply:
Specifically it was added in CPython v3.5.0

Reply:
Ah yes. We may have to add it back once we figure out why it's buggy right now, but let's remove it till someone gets annoyed enough to start looking into that.